projects/odf_transform/odf_transform/vocabulary/CIOOS_Vocabulary_List - Complete_Vocabulary.csv

```diff
@@ -0,0 +1,1480 @@
+Vocabulary,name,Users,MEDS:STATUS,MEDS:CATEGORY_MEANING_E,MEDS:CATEGORY_MEANING_F,MEDS:UNITS_DESCRIPTOR_E,MEDS:UNITS_DESCRIPTOR_F,MEDS:CF_CODE,MEDS:NETCDF_LONG_NAME,MEDS:WMO_CODE_TABLE_ID,MEDS:CONVENTION,BIO:units,BIO:P06 urn,BIO:P06 name,BIO:P01 urn,BIO:P01 name,BIO:CF_CODE,IML:definition,IML:units,long_name,expected_units,scale,standard_name,SDN:P07::id,SDN:P01::id,SDN:P01::urn,SDN:P01::prefLabel,sdn_parameter_urn,sdn_parameter_name,SDN:P06::id,SDN:P06::urn,SDN:P06::prefLabel,sdn_uom_urn,sdn_uom_name,definition,convention,scale_factor,GF3_name,Alternative units,SDN:P01::Alternative Name
```

Comment:
looks like there's a mix of and `|` and `,` separators for `expected_units` - is it supposed to be like that? eg `"kg/m^3|kilogram per metre cube|sigma-theta, kg/m^3|sigma-theta, Kg/m^3"`

Reply:
no the unit has that comma only '|' is a sperator
if comma exist in the string the expected unit needs to be double quoted